poker-app/components/PracticeButtonController.js

```diff
@@ -67,24 +86,129 @@ export default class PracticeButtonController extends Component {
             calls: this.state.calls,
             folds: this.state.folds,
             raises: this.state.raises,
-            tags: this.state.tags
+            tags: this.state.tags,
+            positionStats: this.state.positionStats
         }
         let gamesarr = this.state.gamesArray;
         console.log("LOOOK")
         console.log(gamesarr)
         gamesarr.push(gamesObj)
         let saveObj = {
-            version: "1.0.0",
+            version: "1.0.2",
             games: gamesarr
         }
         return saveObj;
     };
 
-    clearTags(){
+    clearTags() {
         this.setState({
             tag: ''
         })
     }
+    
+     getPosition = (position) =>{
```

Comment:
why is this called getPosition when it is setting position in the state?

Reply:
I called it get position since its the function I pass down to a child component through props, and it gets the value from a child component and sets the state in this component.